src/Symfony/Bundle/WebProfilerBundle/Resources/views/Collector/mailer.html.twig

```diff
@@ -73,165 +73,181 @@
     {% endif %}
 
     {% macro render_transport_details(collector, transport, show_transport_name = false) %}
-        {% set num_emails = collector.events.events(transport)|length %}
+        <div class="card">
+            {% set num_emails = collector.events.events(transport)|length %}
+            {% if num_emails > 1 %}
+                <div class="mailer-email-summary-table-wrapper">
+                    <table class="mailer-email-summary-table">
+                        <thead>
+                            <tr>
+                                <th>#</th>
+                                <th>Subject</th>
+                                <th>To</th>
+                            </tr>
+                        </thead>
+                        <tbody>
+                            {% for event in collector.events.events(transport) %}
+                                <tr class="mailer-email-summary-table-row {{ loop.first ? 'active' }}" data-target="#email-{{ loop.index }}">
```

Comment:
to be accessible, we would need to have a `<button>` in the row, that can be used for keyboard navigation.

Reply:
I've decided to not add that ... because, sadly, the rest of the Symfony Profiler tool is not 100% accessible.

Reply:
well, not being 100% accessible does not mean we should be even less accessible than before...